src/soc_ifc/rtl/soc_ifc_external_reg.rdl

```diff
@@ -226,6 +264,12 @@ reg {
 reg {
     name = "DEBUG & MANUF SERVICE REG";
     desc = "JTAG in debug/manuf mode or SOC can write to this register for ROM/FW defined skips or services; ROM/FW maintains the defintion of these bits.
+            [br]Field decode:
+            [br] [0]     MFG_FLAG_GEN_IDEV_CSR: Enable bit for Caliptra to generate an IDEV CSR
+            [br] [15:1]  MFG_FLAG_RSVD
```

Comment:
There was a recent change to assign bit 1 to " Random Number Generator Unavailable". Would it be possible to add this here as well?

Reply:
Yes. Are bit assignments for this register expected to continue evolving prior to 1.0?

Reply:
Resolved by #283

Reply:
Thanks for adding that! I wouldn't say more changes are expected. But I can't say for sure no other bit assignments would be needed.